packages/radix-ui-themes/src/components/text.tsx

```diff
@@ -28,7 +28,7 @@ const Text = React.forwardRef<TextElement, TextProps>((props, forwardedRef) => {
   } = extractProps(props, textPropDefs, marginPropDefs);
   return (
     <Slot
-      data-accent-color={color || undefined}
```

Comment:
This was a bit of a weird workaround for when callout passed an empty string here, I think the component shouldn't hijack `color=''` into `color={undefined}`